.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - python setup.py build_ext --inplace
```

Comment:
I would avoid in-place build since there might be some files that are not correctly copied.